broker-cli/commands/wallet.js

```diff
@@ -201,13 +202,83 @@ async function networkAddress (args, opts, logger) {
   }
 }
 
+/**
+ * network-status
+ *
+ * ex: `sparkswap wallet network-status`
+ *
+ * @function
+ * @param {Object} args
+ * @param {Object} opts
+ * @param {String} [opts.rpcAddress] broker rpc address
+ * @param {Logger} logger
+ * @return {Void}
+ */
+async function networkStatus (args, opts, logger) {
+  const { market, rpcAddress = null } = opts
+
+  try {
+    const client = new BrokerDaemonClient(rpcAddress)
+    const { baseSymbolCapacities, counterSymbolCapacities } = await client.walletService.getTradingCapacities({market})
+
+    const statusTable = new Table({
+      head: ['', `${baseSymbolCapacities.symbol.toUpperCase()} Capacity`, `${counterSymbolCapacities.symbol.toUpperCase()} Capacity`],
+      colWidths: [14, 24, 24],
+      style: { head: ['gray'] }
+    })
+
+    statusTable.push(['Active', '', ''])
+    statusTable.push([`  Buy ${baseSymbolCapacities.symbol.toUpperCase()}`, formatBalance(baseSymbolCapacities.activeReceiveCapacity, 'active'), formatBalance(counterSymbolCapacities.activeSendCapacity, 'active')])
+    statusTable.push([`  Sell ${baseSymbolCapacities.symbol.toUpperCase()}`, formatBalance(baseSymbolCapacities.activeSendCapacity, 'active'), formatBalance(counterSymbolCapacities.activeReceiveCapacity, 'active')])
+
+    statusTable.push(['Pending', '', ''])
+    statusTable.push([`  Buy ${baseSymbolCapacities.symbol.toUpperCase()}`, formatBalance(baseSymbolCapacities.pendingReceiveCapacity, 'pending'), formatBalance(counterSymbolCapacities.pendingSendCapacity, 'pending')])
+    statusTable.push([`  Sell ${baseSymbolCapacities.symbol.toUpperCase()}`, formatBalance(baseSymbolCapacities.pendingSendCapacity, 'pending'), formatBalance(counterSymbolCapacities.pendingReceiveCapacity, 'pending')])
+
+    statusTable.push(['Inactive', '', ''])
+    statusTable.push([`  Buy ${baseSymbolCapacities.symbol.toUpperCase()}`, formatBalance(baseSymbolCapacities.inactiveReceiveCapacity, 'inactive'), formatBalance(counterSymbolCapacities.inactiveSendCapacity, 'inactive')])
+    statusTable.push([`  Sell ${baseSymbolCapacities.symbol.toUpperCase()}`, formatBalance(baseSymbolCapacities.inactiveSendCapacity, 'inactive'), formatBalance(counterSymbolCapacities.inactiveReceiveCapacity, 'inactive')])
+
+    logger.info(` ${market.bold.white}`)
+    logger.info(statusTable.toString())
+  } catch (e) {
+    logger.error(e)
+  }
+}
+
+/**
+ * Formats the capacities to fixed numbers and colors them if they are greater
+ * than 0
+ *
+ * @function
+ * @param {string} balance
+ * @param {string} status
+ * @return {string} balance formatted with size and color
+ */
+function formatBalance (balance, status) {
+  const fixedBalance = Big(balance).toFixed(16)
+  if (Big(balance).gt(0)) {
+    switch (status) {
```

Comment:
is there a default case here? What is it?

Reply:
No

Reply:
what's the behavior if status is not a value that you're expecting? Seems to me that you should handle it or throw an error or something